### PR TITLE
Testing for #7891 

### PR DESCRIPTION
**Pull request by Tlazypanda**
Testing for #7891 

* [ ] PR is descriptively titled 📑 and links the original issue above 🔗
* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [ ] code is in uniquely-named feature branch and has no merge conflicts 📁
* [ ] screenshots/GIFs are attached 📎 in case of UI updation
* [ ] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by Tlazypanda** [comment]
@jywarren @cesswairimu Not sure how to rebase these commits and just add the last one to unstable server ...also aren't all these commit already merged into master so they would have been first deployed on unstable ...kinda confused with the workflow honestly :confused:  can you help me out? I tried to rebase against unstable but it says everything upto date :sweat_smile:

**Comment by cesswairimu** [comment]
@Tlazypanda on your branch do this `git push -f https://github.com/publiclab/plots2 HEAD:unstable`    this should force push your changes to unstable branch and it will be autodeployed... check the server after about 15min...be sure to notify folks on the channel so that you don't overwrite someone's changes being tested on unstable before you push..

About the workflow question...unstable is independent and not necessarily in sync with master...master is always insync with `stable` but `unstable` is like a sandbox and anyone can force push to it..makes sense?

File: .github/ISSUE_TEMPLATE/--weekly-check-in.md [hunk 1]
```diff
@@ -1,3 +1,12 @@
+---
```
Comment (cesswairimu) [inline .github/ISSUE_TEMPLATE/--weekly-check-in.md:1]: Thanks for noting this...we merged a PR that added this #7949, maybe you could rebase with the master to get this changes...You are doing amazing work  :tada:

Comment (Tlazypanda) [reply .github/ISSUE_TEMPLATE/--weekly-check-in.md:1]: Thanks a ton!! :tada: And thank you for  helping all of us out :clap: :100: and the numerous code reviews :v:

**Comment by Tlazypanda** [comment]
Thanks so much for the detailed explanation @cesswairimu :heart: Will try to do this :v:

**Comment by Tlazypanda** [comment]
@cesswairimu I have pushed the changes :sweat_smile: I tested out seems to work fine ...can you also cross-check? :v:

**Comment by cesswairimu** [comment]
@Tlazypanda just looked at unstable...looks okay

**Comment by Tlazypanda** [comment]
@jywarren can you please check this out and the original pr #7891 Thanks :v:

**Comment by jywarren** [comment]
Will you point this at `main` or is that already part of #7891? Thank you!

**Comment by jywarren** [comment]
It looks super! All good!

**Comment by Tlazypanda** [comment]
Hey @jywarren yes actually the pr #7996 is for the `main` one :sweat_smile:

**Comment by jywarren** [comment]
OK, i left some comments there. Thank you!! Shall we close this one then?

**Comment by Tlazypanda** [comment]
Yes let me go ahead and close this :sweat_smile:

**Pull request closed without merging**